pkgs/by-name/ai/airdrop-cli/package.nix

```
@@ -0,0 +1,36 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  swift,
+  swiftpm,
+}:
+
+stdenv.mkDerivation {
+  pname = "airdrop-cli";
+  version = "unstable-2024-04-13";
+
+  src = fetchFromGitHub {
+    owner = "vldmrkl";
+    repo = "airdrop-cli";
+    rev = "dcdd570c3af3aae509ba7ad9fb26782b427f3a1a";
+    hash = "sha256-7gKKeedRayf27XrOhntu41AMXgxc7fqJRE8Jhbihh3o=";
+  };
+
+  nativeBuildInputs = [
+    swift
+    swiftpm
+  ];
+
+  makeFlags = [
+    "PREFIX=$(out)"
+  ];
+
+  meta = {
+    description = "A macOS CLI for AirDrop written in Swift";
```

```suggestion
    description = "MacOS CLI for AirDrop written in Swift";
```
https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/meta.chapter.md#description-var-meta-description

---

Fixed 👍